package.json

```diff
@@ -137,12 +137,16 @@
     "test-client": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=client test/runner.js",
     "test-server": "NODE_ENV=test NODE_PATH=test:server:client TEST_ROOT=server test/runner.js",
     "test-test": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=test test/runner.js",
+    "test-client:watch": "chokidar '(client|server|test)/**/*.(js|jsx)' -c 'pkill node -f runner.js; npm run test-client ${TEST_PATH}'",
```

Comment:
1. `nodemon` allows you to pass the `-C` flag which would make tests run on change only (not at first launch).   Do you think skipping the initial test-run would be better?
2. Definitely! Imagine something in `client/state/posts` depends on a file in `/client/lib`.  Correctly figuring out exactly which tests to re-run would be a relatively complex task and traversing the require-dependency-tree is a complexity wp-calypso probably doesn't want to get into for such a small gain.